RELEASE_NOTES.md

```diff
@@ -79,6 +119,20 @@
 * [PR-2582] Fix sync bug for tsfiles's directory changed by vitural storage group
 * [ISSUE-2911] Fix The write stream is not closed when executing the command 'tracing off'
 
+# Apache IoTDB 0.11.4
+
+## Bug Fixes
+* IOTDB-1303 Disable group by without aggregation function in select clause
+* IOTDB-1306 Fix insertion blocked caused the deadlock in memory control module 
```

Comment:
I'm not sure if it's necessary. Since this bug is already fixed in 0.11.4, normally we can think it is fixed in 0.12.1 as well. 
By the way, I don't see any duplicated release notes in this doc before.

Reply:
got it.